client/post-editor/test/post-editor.jsx

```diff
@@ -56,6 +56,10 @@ mockery.registerMock( 'my-sites/drafts/draft-list', MOCK_COMPONENT );
 mockery.registerMock( 'lib/layout-focus', {
 	set() {}
 } );
+// TODO: REDUX - add proper tests when whole post-editor is reduxified
```

Comment:
Well, thats true, but following tests are actually rendering the component with all its children.
This needs further deliberation, because I kind of wouldnt call them unit tests.
In any case, this is the test of whole post editor and in my opnion, rewriting it should wait until post editor redux moe is finished.